### DISCUSSION TITLE
is this safe

### DISCUSSION THREAD

**Discussion by Imcooooooool** [AssetRipper/AssetRipper#1396] [Q&A] [answered]
is this actually safe

**Comment by ds5678** [comment]
Do you have a reason to believe it's unsafe?

**Reply by Imcooooooool** [reply]
yea as most of the unity asset rippers got me a virus

**Reply by ds5678** [reply, marked as answer]
You are welcome to upload AssetRipper to a virus checking site like VirusTotal.

**Reply by Imcooooooool** [reply]
hello i have used it and it is thankfully not a virus
